### PR TITLE
fixed python 2 & 3 compatibility

### PR DESCRIPTION
**Pull request by bashu**
make sure that app works in python 3 and 2

PS. now you can prepare and upload new release on PyPi

### CODE REVIEW

File: favicon/models.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 from django.db import models
 from django.conf import settings
 from django.core.files.uploadedfile import InMemoryUploadedFile
 
 from PIL import Image
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO
+from six import BytesIO
```
Comment (philippeowagner) [inline favicon/models.py]: @bashu could you please use the [compat](https://github.com/arteria/django-compat/) methods, in this case https://github.com/arteria/django-compat/blob/master/compat/__init__.py#L406

Thanks.

**Comment by bashu** [comment]
fixed as suggested by @philippeowagner

**Comment by philippeowagner** [comment]
Great @bashu :+1: 
Should I pack and push to PyPI?

**Comment by bashu** [comment]
would be awesome!

**Comment by philippeowagner** [comment]
@bashu there you go [django-favicon-plus/0.0.2](https://pypi.python.org/pypi/django-favicon-plus/0.0.2) Thanks for your improvements!

**Pull request merged**